src/textual/layout.py

```diff
@@ -217,6 +218,18 @@ def get_style_at(self, x: int, y: int) -> Style:
         return Style.null()
 
     def get_widget_region(self, widget: Widget) -> Region:
+        """Get the Region of a Widget contained in this Layout.
+
+        Args:
+            widget: The Widget in this layout you wish to know the Region of.
```

Comment:
Could you add the type `(Widget)` to the arg. I know its obvious in context, but we will have auto-docs at some point.